lib/shared/types.ts

```diff
@@ -84,6 +84,11 @@ export interface SystemConfig {
   };
   llms: {
     sagemaker: SupportedSageMakerModels[];
+    sagemakerSchedule?: {
+      enabled?: boolean;
```

Comment:
I know i commented on it way at the start, but again here - is there a scenario where enabled is ommited but you would still pass the schedules?